project-base/tests/App/Functional/Twig/FilesystemLoaderTest.php

```diff
@@ -4,19 +4,20 @@
 
 namespace Tests\App\Functional\Twig;
 
-use ReflectionClass;
 use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Domain\Multidomain\Twig\FilesystemLoader;
 use Shopsys\FrameworkBundle\Component\Setting\Setting;
 use Tests\App\Test\FunctionalTestCase;
+use Twig\Environment;
 
 class FilesystemLoaderTest extends FunctionalTestCase
 {
-    public function testLoadingMultiDesignTemplate()
+    public function testLoadingMultiDesignTemplate(): void
     {
         $twigFilesystemLoader = $this->getContainer()->get('twig.loader.native_filesystem');
```

Comment:
Doesn't mark EA plugin `$this->getContainer()` with suggestion `self::getContainer()`? It is now static